cmake/iree_aie_xrt.cmake

```diff
@@ -176,7 +175,10 @@ foreach(_core_lib IN LISTS _core_libs)
                              ${IREE_XRT_SOURCE_DIR}/runtime_src)
   target_include_directories(${_core_lib} SYSTEM PUBLIC
                              ${IREE_XRT_SOURCE_DIR}/runtime_src/core/common/elf)
-  target_compile_definitions(${_core_lib} PUBLIC -DBOOST_BIND_GLOBAL_PLACEHOLDERS)
+  target_compile_definitions(${_core_lib}
+                             PUBLIC
+                             -DBOOST_BIND_GLOBAL_PLACEHOLDERS
+                             $<$<CONFIG:Debug>:-DXRT_VERBOSE>)
```

Comment:
Is this for debugging purposes?

Reply:
Yes

Reply:
Should we enable it by default?

Reply:
Nah it's too "loud".

Reply:
Sorry, I was assuming it was enabled by default now. If it's not, that's good